### PR TITLE
Configure chatbot with script

### PR DESCRIPTION
**Pull request by papajulio**

### CODE REVIEW

File: chatbot-configuration/load_data.py [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+
+    text = dialogflow.types.Intent.Message.Text(text=messages)
+    message = dialogflow.types.Intent.Message(text=text)
+    return message
+
+
+def _get_training_phrases_from_json(intent):
+    training_phrases = []
+    if 'training_phrases' in intent:
+        for training_phrase in intent['training_phrases']:
```
Comment (figarocorso) [inline chatbot-configuration/load_data.py]: An alternative for previous comment could be:
```python
for training_phrase in intent.get('training_phrases', []):
```

This kind of improvements (`if` or `for` ones) can be done at some other places, please, take a look if you think the improvement makes sense.

File: chatbot-configuration/load_data.py [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+def create_or_update_intent(project_id, intent):
+    logger.info("Creating or updating intent {}...".format(intent['name']))
+    if _get_intent_ids(project_id, intent['name']):
+        if _get_parent_followup_intent_name_from_json(project_id, intent):
+            delete_intent(project_id, _get_intent_ids(project_id, intent['name'])[0])
+            create_intent(project_id, intent)
+        else:
+            update_intent(project_id, intent)
+    else:
+        create_intent(project_id, intent)
```
[the remark this answers is not in the document]
Comment (papajulio) [reply chatbot-configuration/load_data.py]: I don't like to use return too much, but I see your point about the if-else nested, I will change this hopefully you wold like it more

File: chatbot-configuration/load_data.py [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+    entity_types = entity_types_client.list_entity_types(parent)
+    entity_type_names = [
+        entity_type.name for entity_type in entity_types
+        if entity_type.display_name == display_name]
+
+    entity_type_ids = [
+        entity_type_name.split('/')[-1] for entity_type_name
+        in entity_type_names]
+
+    return entity_type_ids
```
[the remark this answers is not in the document]
Comment (papajulio) [reply chatbot-configuration/load_data.py]: This code it's not mine so I didn't review it too carefully it's a copy and paste from the dialogflow examples, you can check it here: https://github.com/dialogflow/dialogflow-python-client-v2/blob/master/samples/entity_management.py

File: chatbot-configuration/README.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+This contains the dialogflow configuration of the chatbot
+
+Follow this steps to configure your pgoogle project so you can upload/edit the chatbot:
+* https://github.com/dialogflow/dialogflow-python-client-v2#before-you-begin
+
+Execute the script in the following way to create or update the chatbot:
+
+```
+GOOGLE_APPLICATION_CREDENTIALS={PATH_TO_JSON_FILE_WITH_THE_GOOGLE_CREDENTIALS} python load_data.py --project-id {PROJECT_ID}
```
[the remark this answers is not in the document]
Comment (papajulio) [reply chatbot-configuration/README.md]: I prefer to leave as it is, as in the official examples of the library are using the same approach and it seems to me a quite standard way of handling the auth (in AWS libraries similar is being used)

**Comment by papajulio** [comment]
@figarocorso handled all your comments, thanks for the  thoughtful review

File: chatbot-configuration/load_data.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (figarocorso) [reply chatbot-configuration/load_data.py]: Sure, I can live with that :+1:

**Pull request merged**